### PR TITLE
Fix: remove duplicate content type header

### PR DESCRIPTION
**Pull request by thewahome**
## Overview

Removes duplicate content-type headers passed in from the sample queries

### Demo

image

## Testing Instructions

* Select send an email from the sidebar
* Click run query
* Message will be sent

### CODE REVIEW

File: src/app/services/actions/query-action-creator-util.ts [hunk 1]
```diff
@@ -103,9 +103,13 @@ const makeRequest = (httpVerb: string, scopes: string[]): Function => {
 
     if (query.sampleHeaders) {
       query.sampleHeaders.forEach(header => {
-        if (header.name !== '' && header.value !== '') {
-          sampleHeaders[header.name] = header.value;
+        if (header.name.toLowerCase() === 'content-type') {
```
Comment (MIchaelMainer) [inline src/app/services/actions/query-action-creator-util.ts:106]: We should check that content-type value is set.

**Comment by MIchaelMainer** [comment]
Fixes #533

File: src/app/services/actions/query-action-creator-util.ts [hunk 2]
```diff
@@ -103,9 +103,13 @@ const makeRequest = (httpVerb: string, scopes: string[]): Function => {
...
+          return;
         }
+        if (header.name === '' && header.value === '') {
```
Comment (MIchaelMainer) [inline src/app/services/actions/query-action-creator-util.ts:109]: No use in setting a header if either value is empty string. I think this should be an **or** condition.

File: src/app/services/actions/query-action-creator-util.ts (hunk 1 above)
Comment (MIchaelMainer) [reply src/app/services/actions/query-action-creator-util.ts:106]: Relying on the graph client to set the content type. Please add comment that content-type header is being set by Graph JavaScript client.

File: src/app/services/actions/query-action-creator-util.ts (hunk 2 above)
Comment (thewahome) [reply src/app/services/actions/query-action-creator-util.ts:109]: I actually think this check is redundant now

**Pull request merged**